### PR TITLE
Backspace text area fix cals 5637

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-5637

#### Brief feature description
- fix for backspace bug in body of water and directions to house that was intorudeced as a part of fixing the ie backspace issue. 

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- 82.56% Statements 
 69.06% Branches 
 78.19% Functions
 83.15% Lines

### CODE REVIEW

**Comment by Nasir2712** [comment]
can you update the test case in rspec?

**Comment by nasar-cambria** [comment]
Except the above comment everything else looks good.

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -20,11 +20,14 @@ ReactRailsUJS.useContext(componentRequireContext)
...
   let keyCode = e.which
-  if (document.activeElement === document.body ||
+  if ((document.activeElement === document.body ||
       document.activeElement.tagName === 'SELECT' ||
       document.activeElement.tagName === 'BUTTON' ||
       document.activeElement.tagName === 'A' ||
-      document.activeElement.type !== 'text'
+      document.activeElement.type === 'radio' ||
+      document.activeElement.type === 'checkbox') &&
+     (document.activeElement.type !== 'text' ||
```
Comment (Nasir2712) [inline app/javascript/packs/application.js]: radio and checkbox are input types as well. document.activeElement.type !== 'text' actually takes care of them. I don't think you need to add a check for radio or checkbox again.

Comment (lauherkcambria) [reply app/javascript/packs/application.js]: unfortunately, we need to. not adding those two lines makes the return to previous page issue in IE occur if the user hits the backspace on either a checkbox or radio. Thus, these two lines are necessary.

**Pull request merged**